src/main/java/li/cil/tis3d/api/CommonAPI.java

```diff
@@ -0,0 +1,17 @@
+package li.cil.tis3d.api;
+
+import net.minecraft.item.ItemGroup;
+
+public final class CommonAPI {
+    /**
+     * The ID of the mod, i.e. the internal string it is identified by.
+     */
+    public static final String MOD_ID = "tis3d";
```

Comment:
I haven't quite understood why this is part of the API package; is this for querying whether the mod is loaded at runtime?

Reply:
This is/was mainly for stuff in the API needing the mod id; e.g. `ResourceLocation`/`Identifier` stuff. Just to not having to hardcode it everywhere. Could e.g. be used [here](https://github.com/MightyPirates/TIS-3D/blob/master-MC1.16-fabric/src/main/java/li/cil/tis3d/api/util/RenderLayerAccess.java#L13).